files/ru/web/api/webrtc_api/adapter.js/index.md

```diff
@@ -6,7 +6,7 @@ tags:
 translation_of: Web/API/WebRTC_API/adapter.js
 ---
 
-{{WebRTCSidebar}}
+{{DefaultAPISidebar("WebRTC")}}
```

Comment:
This file is removed in the English, but this might be pending a Sync PR landing here